moha/toml_tools/defaults.toml

```diff
@@ -2,7 +2,7 @@
 outdir = "."
 save_integrals = false
 integral_format = "fcidump"
-prefix = ''
+prefix = 'ModelHamiltonian'
```

Comment:
I left prefix to be empty string, so that I we can set it to the name of the model. I'll change it back if there is no functionality tied on this